src/main/scala/info/bethard/timenorm/formal/Types.scala

```diff
@@ -26,7 +26,7 @@ case class FractionalNumber(number: Int, numerator: Int, denominator: Int) exten
   val isDefined = true
 }
 
-case class VagueNumber(description: String) extends Number {
+case class VagueNumber(description: Option[String]) extends Number {
```

Comment:
Don't make `VagueNumber` take an `Option[String]`. Rather, in your scorer where the value of the `VagueNumber` doesn't matter, just pass some meaningless value. Something like `entity.text.getOrElse("")`.